.husky/.gitattributes

```diff
@@ -0,0 +1 @@
+pre-commit eol=lf
```

Comment:
https://github.com/microsoft/BotFramework-WebChat/issues/3842

Can you add the above to list of fixed items? (Skipping changelog is fine - just in PR)

Reply:
No worries, I want to run the pipeline multiple times to catch more test reliability issues. It's like whack-a-mole.